src/com/example/supabase.cljs

```diff
@@ -24,3 +26,28 @@
       .-auth
       .user
       (js->clj :keywordize-keys true)))
+
+;; .from('cities')
+;;   .insert([
+;;     { name: 'The Shire', country_id: 554 },
+;;     { name: 'Rohan', country_id: 555 },
+;;   ])
+(defn insert-response [{:keys [response/total user/id]}]
+  (go
+    (try
+      (<p! (-> @client
+               (.from "response_scores")
+               (.insert #js [#js {"total" total "user_id" id}])))
+      (catch js/Error err (js/console.log (ex-cause err))))))
```

Comment:
this is equivalent to calling:

```
This is equivalent to calling

```js
try {
  return await client.from("response_scores").insert({"total": total, "user_id": id})
} catch(err) {
  console.log(err.cause)
}
```